Library/Homebrew/extend/os/linux/install.rb

```diff
@@ -13,6 +13,8 @@ module Install
     ].freeze
 
     def symlink_ld_so
+      return if ENV["HOMEBREW_SYSTEM"] == "Cygwin"
```

Comment:
This should be in `extend/os/cygwin` instead.

Reply:
Yes, my initial attempts were based on the information from this old request: https://github.com/Homebrew/linuxbrew-core/issues/3880. But have fixed that now.